web_responsive/static/src/legacy/scss/web_responsive.scss

```diff
@@ -0,0 +1,380 @@
+                display: block;
+                span {
+                    display: block;
+                    max-width: 100%;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    overflow: hidden;
+                    width: initial;
+                    &:active {
+                        white-space: normal;
+                    }
+                }
+            }
+        }
+
+        @include media-breakpoint-down(sm) {
+            min-width: auto;
+
+            // More buttons border
+            .oe_button_box {
+                .o_dropdown_more {
+                    button:last-child {
+                        border-right: 1px solid $gray-400 !important;
+                    }
+                }
+            }
+
+            .oe_button_box + .oe_title,
+            .oe_button_box + .oe_avatar + .oe_title {
+                width: 100%;
+            }
+
+            // Avoid overflow on modals
+            .o_form_sheet {
+                min-width: auto;
+            }
+
+            // Render website inputs properly in phones
+            .o_group .o_field_widget.o_text_overflow {
+                // Overrides another !important
+                width: auto !important;
+            }
+
+            // Full width in form sheets
+            .o_form_sheet,
+            .o_FormRenderer_chatterContainer {
+                min-width: auto;
+                max-width: 98% !important;
+            }
+
+            // Settings pages
+            .app_settings_block {
+                .row {
+                    margin: 0;
+                }
+            }
+
+            .o_FormRenderer_chatterContainer {
+                padding-top: initial;
+
+                // Display send button on small screens
+                .o_Chatter_composer {
+                    &.o-has-current-partner-avatar {
+                        grid-template-columns: 0px 1fr;
+                        padding: 1rem 1rem 1.5rem 1rem !important;
+                    }
+
+                    .o_Composer_sidebarMain {
+                        display: none;
+                    }
+                }
+            }
+        }
+    }
+
+    //No content message improvements on mobile
+    @include media-breakpoint-down(md) {
+        .o_view_nocontent {
+            top: 80px;
+        }
+        .o_nocontent_help {
+            box-shadow: none;
+        }
+        .o_sample_data_disabled {
+            display: none;
+        }
+    }
+}
+
+// Sticky Header & Footer in List View
+.o_list_view {
+    .table-responsive {
+        overflow: visible;
+        .o_list_table {
+            // th & td are here for compatibility with chrome
+            thead tr:nth-child(1) th {
+                position: sticky !important;
+                top: 0;
+                z-index: 999;
+            }
+            thead tr:nth-child(1) th {
+                background-color: $o-list-footer-bg-color;
```

Comment:
```suggestion
                background-color: var(--ListRenderer-thead-bg-color);
```

Reply:
To use the v16.0 CE default header bg color, which looks great i think.

Reply:
Updated this